vnext/Microsoft.ReactNative/IViewManager.idl

```diff
@@ -92,7 +92,7 @@ namespace Microsoft.ReactNative
   [webhosthidden]
   DOC_STRING(
     "Enables view managers to track when views are"
```

Comment:
this still is missing a space after are (the string in line 95 gets concatenated to line 94